assignment.js

```diff
@@ -112,7 +124,46 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
-
+let inputA = 'scissors'
+let inputB = 'paper'
+
+inputA = inputA.toLowerCase();
+inputB = inputB.toLowerCase();
+
+function inputCheck() {
```

Comment:
@studiosok nice use a function to implement error handling (it wasn't required in this, but great job covering this use case)